scripts/west_commands/runners/canopen_program.py

```diff
@@ -87,7 +87,7 @@ def do_add_parser(cls, parser):
         parser.set_defaults(confirm=True)
 
     @classmethod
-    def create(cls, cfg, args):
+    def do_create(cls, cfg, args):
```

Comment:
just curious, is there some Python best practices that favors `do_create` over  `create` ?

Reply:
Forget my comment, I later discovered you also have `create` and also noticed:
```
a do_create(), a new ZephyrBinaryRunner abstract class method,
and make create() itself concrete.
```